kfdefs/overlays/moc/smaug/opf-datacatalog/user-buckets/user-dtuch.enc.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
```

Comment:
User bucket definitions has changed, please rebase and copy from https://github.com/operate-first/apps/pull/842 no ksops needed anymore.

Reply:
this should be addressed.